src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
```

Comment:
this is default flex property, so u don't have to write it, but it's okey

Reply:
Deleted